src/driver/postgres/PostgresDriver.ts

```diff
@@ -297,6 +297,13 @@ export class PostgresDriver implements Driver {
         materializedHint: true,
     }
 
+    /**
+     * Is true if the native driver is used. The native driver does not support
+     * streaming, so you can use this property to disable streaming in your
+     * application.
+     */
+    isNative = false
```

Comment:
Added this for runtime detection of features, that way I can disable the tests that use features not available in the native driver.